src/codegen.cpp

```diff
@@ -10243,9 +10243,6 @@ extern "C" void jl_init_llvm(void)
 #if defined(JL_USE_INTEL_JITEVENTS) || \
     defined(JL_USE_OPROFILE_JITEVENTS) || \
     defined(JL_USE_PERF_JITEVENTS)
-#ifdef JL_USE_JITLINK
-#pragma message("JIT profiling support (JL_USE_*_JITEVENTS) not yet available on platforms that use JITLink")
-#else
```

Comment:
My understanding is that this is still true in llvm 18 (and at the moment we support up to 17)?  https://github.com/llvm/llvm-project/pull/83957 isn't in any stable version yet (presumably will be in LLVM 19).

Reply:
It's likely this backports quite cleanly to 18 though

Reply:
It is no longer relevant though, since they are (increasingly) available with each new version of LLVM